src/js/coblocks-google-maps.js

```diff
@@ -1,1064 +1,1057 @@
+			stylers: [
+				{
+					color: '#023e58',
+				},
+			],
+		},
+		{
+			elementType: 'geometry',
+			featureType: 'poi',
+			stylers: [
+				{
+					color: '#283d6a',
+				},
+			],
+		},
+		{
+			elementType: 'labels.text.fill',
+			featureType: 'poi',
+			stylers: [
+				{
+					color: '#6f9ba5',
+				},
+			],
+		},
+		{
+			elementType: 'labels.text.stroke',
+			featureType: 'poi',
+			stylers: [
+				{
+					color: '#1d2c4d',
+				},
+			],
+		},
+		{
+			elementType: 'geometry.fill',
+			featureType: 'poi.park',
+			stylers: [
+				{
+					color: '#023e58',
+				},
+			],
+		},
+		{
+			elementType: 'labels.text.fill',
+			featureType: 'poi.park',
+			stylers: [
+				{
+					color: '#3C7680',
+				},
+			],
+		},
+		{
+			elementType: 'geometry',
+			featureType: 'road',
+			stylers: [
+				{
+					color: '#304a7d',
+				},
+			],
+		},
+		{
+			elementType: 'labels.text.fill',
+			featureType: 'road',
+			stylers: [
+				{
+					color: '#98a5be',
+				},
+			],
+		},
+		{
+			elementType: 'labels.text.stroke',
+			featureType: 'road',
+			stylers: [
+				{
+					color: '#1d2c4d',
+				},
+			],
+		},
+		{
+			elementType: 'geometry',
+			featureType: 'road.highway',
+			stylers: [
+				{
+					color: '#2c6675',
+				},
+			],
+		},
+		{
+			elementType: 'geometry.stroke',
+			featureType: 'road.highway',
+			stylers: [
+				{
+					color: '#255763',
+				},
+			],
+		},
+		{
+			elementType: 'labels.text.fill',
+			featureType: 'road.highway',
+			stylers: [
+				{
+					color: '#b0d5ce',
+				},
+			],
+		},
+		{
+			elementType: 'labels.text.stroke',
+			featureType: 'road.highway',
+			stylers: [
+				{
+					color: '#023e58',
+				},
+			],
+		},
+		{
+			elementType: 'labels.text.fill',
+			featureType: 'transit',
+			stylers: [
+				{
+					color: '#98a5be',
+				},
+			],
+		},
+		{
+			elementType: 'labels.text.stroke',
+			featureType: 'transit',
+			stylers: [
+				{
+					color: '#1d2c4d',
+				},
+			],
+		},
+		{
+			elementType: 'geometry.fill',
+			featureType: 'transit.line',
+			stylers: [
+				{
+					color: '#283d6a',
+				},
+			],
+		},
+		{
+			elementType: 'geometry',
+			featureType: 'transit.station',
+			stylers: [
+				{
+					color: '#3a4762',
+				},
+			],
+		},
+		{
+			elementType: 'geometry',
+			featureType: 'water',
+			stylers: [
+				{
+					color: '#0e1626',
+				},
+			],
+		},
+		{
+			elementType: 'labels.text.fill',
+			featureType: 'water',
+			stylers: [
+				{
+					color: '#4e6d70',
+				},
+			],
+		},
+	];
 
-		return GMapStyles[ key ];
-	},
+	return GMapStyles[ key ];
 };
 
-jQuery( document ).ready( function() {
-	coblocksMap.init();
+// Disable note: This rule is only meant to be used for React components
+// See https://github.com/WordPress/gutenberg/pull/26810
+// eslint-disable-next-line @wordpress/no-global-event-listener
+document.addEventListener( 'DOMContentLoaded', function() {
+	const elements = document.getElementsByClassName( 'wp-block-coblocks-map' );
```

Comment:
I guess the const here might be a little too verbose. I think this is readable enough `renderMap( document.getElementsByClassName( 'wp-block-coblocks-map' ) );`

Unless we want to make sure there are elements on the page before calling renderMap.